interpreter/interpreter_test.go

```diff
@@ -30,6 +30,51 @@ import (
 	exprpb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
 )
 
+func TestCompleteInterpreter_ConditionalExpr(t *testing.T) {
+	// a ? b < 1.0 : c == ["hello"]
+	program := NewCompleteProgram(
+		test.Conditional.Expr,
+		test.Conditional.Info(t.Name()))
+
+	interpretable := interpreter.NewInterpretable(program)
+	result, state := interpretable.Eval(
+		NewActivation(map[string]interface{}{
+			"a": true,
+			"b": 0.999,
+			"c": types.NewStringList([]string{"hello"})}))
```

Comment:
Would you mind adding a comment about why the test for the value of `c` is being done?

Reply:
Comments added~ Please let me know if it is sufficient~